e2e/fixtures/child.js

```diff
@@ -0,0 +1,58 @@
+module.exports = [
```

Comment:
We could rename file to be `children` since we have more than one child there.

Reply:
Can I rename respondent to respondents while i'm here?